package.json

```diff
@@ -34,7 +35,8 @@
     "mocha": "6.1.4"
```

Comment:
could also remove `exaca` and `async` .. we can pull them back in if we need them for scripts later